### PR TITLE
Fix schedule instructions for "prefect deploy"

### LINKED ISSUE #12094: Docs on multiple schedules in `prefect deploy` YAML are wrong
### First check

- [X] I added a descriptive title to this issue.
- [X] I used GitHub search to find a similar request and didn't find it 😇

### Describe the issue

This section says that the YAML produced by `prefect deploy` supports multiple schedules: https://docs.prefect.io/latest/concepts/schedules/?h=schedules#creating-schedules-through-a-prefectyaml-files-deployments-schedules-section

However, it doesn't. Only the YAML produced by `prefect deployment build` supports multiple schedules.

See: #12092 

### Describe the proposed change

Update the example to explain which YAML file supports multiple schedules and that we are rolling out multiple schedules support to `prefect deploy` soon.

### Additional context

_No response_

### PR DESCRIPTION
**Pull request by abrookins**
The instructions for using schedules with `prefect deploy` say that the command supports multiple schedules. That's not true yet. Currently, support exists in the UI, CLI via `prefect deployment schedule` commands, the `Deployment` class, and the YAML file created for block-based deployments via `prefect deployment build`.

fixes #12094 

### Checklist

- [x] This pull request references any related issue by including "closes `<link to issue>`"
	- If no issue exists and your change is not a small fix, please [create an issue](https://github.com/PrefectHQ/prefect/issues/new/choose) first.
- [x] This pull request includes tests or only affects documentation.
- [x] This pull request includes a label categorizing the change e.g. `maintenance`, `fix`, `feature`, `enhancement`, `docs`.
  

For documentation changes:

- [ ] This pull request includes redirect settings in `netlify.toml` for files that are removed or renamed.

For new functions or classes in the Python SDK:

- [ ] This pull request includes helpful docstrings.
- [ ] If a new Python file was added, this pull request contains a stub page in the Python SDK docs and an entry in `mkdocs.yml` navigation.

### CODE REVIEW

File: docs/concepts/schedules.md [hunk 1]
```diff
@@ -31,19 +31,28 @@ There are several recommended ways to create a schedule for a deployment:
 - Via a the `cron`, `interval`, or `rrule` parameters if building your deployment via the [`serve` method](/concepts/flows/#serving-a-flow) of the `Flow` object or [the `serve` utility](/concepts/flows/#serving-multiple-flows-at-once) for managing multiple flows simultaneously
 - If using [worker-based deployments](/concepts/work-pools/)
   - Through the interactive `prefect deploy` command
-  - With the `deployments` -> `schedules` section of the `prefect.yaml` file )
+  - With the `deployments` -> `schedule` section of the `prefect.yaml` file )
+
+## Support for multiple schedules
+
+We are currently rolling out support for multiple schedules per deployment. You can now assign multiple schedules to Deployments in the Prefect UI, the CLI via `prefect deployment schedule` commands, the `Deployment` class, and in [block-based deployment](/concepts/deployments/#block-based-deployments) YAML files.
```
Comment (discdiver) [inline docs/concepts/schedules.md:38]:
```suggestion
We are currently rolling out support for multiple schedules per deployment. You can now assign multiple schedules to deployments in the Prefect UI, the CLI via `prefect deployment schedule` commands, the `Deployment` class, and in [block-based deployment](/concepts/deployments/#block-based-deployments) YAML files.
```

File: docs/concepts/schedules.md [hunk 2]
```diff
@@ -31,19 +31,28 @@ There are several recommended ways to create a schedule for a deployment:
...
+
+Support for multiple schedules in `flow.serve`, `flow.deploy`, `serve`, and `prefect deploy` will arrive in Q1 2024.
```
Comment (discdiver) [inline docs/concepts/schedules.md:40]:
```suggestion
Support for multiple schedules in `flow.serve`, `flow.deploy`, `serve`, and `prefect deploy` will arrive soon.
```

File: docs/concepts/schedules.md [hunk 3]
```diff
@@ -31,19 +31,28 @@ There are several recommended ways to create a schedule for a deployment:
...
 
 ## Creating schedules through the UI
 
 You can add a schedule on the **Deployments** tab of the UI.
 
-Under **Schedules** click the **+** button.
+Under **Schedule** select the **Add** button.
```
Comment (discdiver) [inline docs/concepts/schedules.md:46]:
```suggestion
Under **Schedules** select the **+ Schedule** button.
```

File: docs/concepts/schedules.md [hunk 4]
```diff
@@ -31,19 +31,28 @@ There are several recommended ways to create a schedule for a deployment:
...
+If you don't see the **Schedule** section, expand the three dot menu on the top right of the page.
```
Comment (discdiver) [inline docs/concepts/schedules.md:47]:
```suggestion
If you don't see the **Schedule** section, increase the width of the page.
```

File: docs/concepts/schedules.md [hunk 5]
```diff
@@ -31,19 +31,28 @@ There are several recommended ways to create a schedule for a deployment:
...
+
 Then select **Interval** or **Cron** to create a schedule.
 
 ![Prefect UI with Interval button selected](/img/ui/interval-schedule.png)
 
 Once a schedule has been created, a number of scheduled flow runs will be visible in the UI.
 The schedule is viewable in human-friendly text on the **Deployments** page.
+You can edit the schedule by selecting the **Edit** button on the **Deployment** page.
```
Comment (discdiver) [inline docs/concepts/schedules.md:55]:
```suggestion
You can edit a schedule by selecting **Edit** from the three dot menu next to a schedule on a **Deployment** page.
```

File: docs/concepts/schedules.md [hunk 6]
```diff
@@ -31,19 +31,28 @@ There are several recommended ways to create a schedule for a deployment:
...
+
+## Support for multiple schedules
+
+We are currently rolling out support for multiple schedules per deployment. You can now assign multiple schedules to Deployments in the Prefect UI, the CLI via `prefect deployment schedule` commands, the `Deployment` class, and in [block-based deployment](/concepts/deployments/#block-based-deployments) YAML files.
+
+Support for multiple schedules in `flow.serve`, `flow.deploy`, `serve`, and `prefect deploy` will arrive in Q1 2024.
 
 ## Creating schedules through the UI
 
 You can add a schedule on the **Deployments** tab of the UI.
```
Comment (discdiver) [inline docs/concepts/schedules.md:44]:
```suggestion
You can add schedules on a **Deployment** page in the UI.
```

**Review by discdiver — CHANGES_REQUESTED** [review]
Thank you! Updating UI instructions to what I'm seeing in prod.

File: docs/concepts/schedules.md [hunk 7]
```diff
@@ -22,28 +22,13 @@ Prefect Cloud can also schedule flow runs through event-driven [automations](/co
...
-You can add a schedule on the **Deployments** tab of the UI.
-
-Under **Schedules** click the **+** button.
-Then select **Interval** or **Cron** to create a schedule.
-
-![Prefect UI with Interval button selected](/img/ui/interval-schedule.png)
-
-Once a schedule has been created, a number of scheduled flow runs will be visible in the UI.
-The schedule is viewable in human-friendly text on the **Deployments** page.
+    Support for multiple schedules in `flow.serve`, `flow.deploy`, `serve`, and [worker-based deployments](/concepts/work-pools/) with `prefect deploy` will arrive soon.
```
Comment (discdiver) [inline docs/concepts/schedules.md:31]: Serve support PR has at least one approval, so likely to be in this release, too. #12107

File: docs/concepts/schedules.md [hunk 8]
```diff
@@ -22,28 +22,13 @@ Prefect Cloud can also schedule flow runs through event-driven [automations](/co
 
 Schedules tell the Prefect API how to create new flow runs for you automatically on a specified cadence.
 
-You can add schedules to any flow [deployment](/concepts/deployments/).
+You can add a schedule to any flow [deployment](/concepts/deployments/).
```
Comment (discdiver) [inline docs/concepts/schedules.md:25]:
```suggestion
You can add a schedule to any [deployment](/concepts/deployments/).
```

**Review by discdiver — APPROVED** [review]
Nice! Made a note and one little edit.

File: docs/concepts/schedules.md (hunk 7 above)
Comment (abrookins) [reply docs/concepts/schedules.md:31]: Oh, good point!

Comment (abrookins) [reply docs/concepts/schedules.md:31]: I'm going to wait and we can update that if we do indeed ship `flow.serve` support, which is very likely given the PR... but not merged yet.

**Pull request merged**